### ISSUE TITLE
UI Localization

### ISSUE THREAD

**Issue by escattone** [mdn/yari#1197]
We need to define and implement how we're going to do localization of UI strings. Can we use fluent as part of this? Do the following sub-tasks still make sense:
- [ ] Gather strings from code
- [ ] Integration with Pontoon

**Comment by SphinxKnight** [comment]
@escattone is it possible to add the localization label on this issue please?

**Comment by escattone** [comment]
@schalkneethling Yes, we should keep this open. It may be a number of months before it's addressed, but I think it's important to address this eventually as time allows.

**Comment by SphinxKnight** [comment]
Given https://blog.mozilla.org/en/mozilla/mozilla-webvision-future-of-web/ "Provide a first-class experience for non-English-speakers", how could we help / achieve MDN UI's localization? (which existed under the previous platform)

@HerminaC who would be the best interlocutor to set things in motion? Given the impact on the codebase would PRs be accepted regarding strings' extraction and the like?

**Comment by schalkneethling** [comment]
> Given https://blog.mozilla.org/en/mozilla/mozilla-webvision-future-of-web/ "Provide a first-class experience for non-English-speakers", how could we help / achieve MDN UI's localization? (which existed under the previous platform)
> 
> @HerminaC who would be the best interlocutor to set things in motion? Given the impact on the codebase would PRs be accepted regarding strings' extraction and the like?

This is something we have been actively discussing. We have not reached full consensus yet, but if you have any suggestions/recommendation, please do let us know. Thanks, @SphinxKnight

**Comment by SphinxKnight** [comment]
Referencing #6282 here

**Comment by caugner** [comment]
We can consider this resolved, as [Fred](https://github.com/mdn/fred) now supports localization.
